l10n_th_withholding_tax_cert/models/withholding_tax_cert.py

```diff
@@ -73,6 +84,20 @@ class WithholdingTaxCert(models.Model):
         copy=False,
         ondelete="restrict",
     )
+    company_id = fields.Many2one(
```

Comment:
To correctly deal with company_dependent.
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration
![image](https://user-images.githubusercontent.com/1973598/94885575-c1554e00-049a-11eb-885f-2fea49c099ed.png)